.github/workflows/test.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-20.04, ubuntu-22.04]
+        os: [ubuntu-22.04, ubuntu-24.04]
```

Comment:
I think we were using Ubuntu 20 so we could test cgroups v1

Reply:
Ubuntu 20.04 stopped being available last month, so we can't use that runner anymore https://github.com/actions/runner-images?tab=readme-ov-file#available-images

Reply:
Oh well. I suppose we could set up a custom runner but I don't think that's for it just for v1 coverage.

Reply:
Yeah probably not worth the headache of setting up, but sad it'll be untested now though. In that case I'll just update this CI to run on ubuntu-latest instead

Reply:
https://github.com/python-discord/snekbox/pull/236/commits/3027f4b9cecab1f3f4e05eeefdcbd33593e54745